lib/web/question_router.ts

```diff
@@ -1494,6 +1494,17 @@ async function getQuestionsUserCreatedOrForecastedOnOrIsSharedWith(
                       },
                     },
                   },
+                  {
+                    tags: {
+                      some: {
+                        name: {
+                          contains: input.extraFilters.searchString,
+                          mode: "insensitive",
+                        },
+                        userId: ctx.userId, 
```

Comment:
Should probably be `userIdIfAuthed` (defined above)

Reason: if user is logged out, ctx.userId == undefined, and so prisma doesn't filter based on it. More info: https://www.prisma.io/docs/orm/prisma-client/special-fields-and-types/null-and-undefined